sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/AppendBlobAPITest.groovy

```diff
@@ -161,6 +161,7 @@ class AppendBlobAPITest extends APISpec {
 
     def "Append block defaults"() {
         setup:
+
```

Comment:
Is this change needed?

Reply:
No, will revert. Will update tests in another commit.